qemu/tests/cfg/balloon_memhp.cfg

```diff
@@ -22,7 +22,7 @@
     balloon = balloon0
     balloon_dev_devid = balloon0
     balloon_dev_add_bus = yes
-    threshold = 0.12
+    threshold = 0.19
```

Comment:
Would you pls run more times with different guests? 0.19 doesn't look so reliable to me.

Reply:
More comments are added in the bz.